content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all the external services by default
```

Comment:
all the services -> all services

Reply:
Fixed in   https://github.com/istio/istio.io/pull/3140/commits/ba06575d7865a462b6c92771774ebadaf4f2aa0b